XenAdmin/MainWindow.ja.resx

```diff
@@ -2445,6 +2445,12 @@
   <data name="DestroyStorageRepositoryToolStripMenuItem.Text" xml:space="preserve">
     <value>破棄(&amp;O)</value>
   </data>
+  <data name="ConvertToThinStorageRepositoryToolStripMenuItem.Size" type="System.Drawing.Size, System.Drawing">

```

Comment:
These controls have been removed in PR #1728, where we removed all the thin provisioning code. Please make sure that we are not regressing that commit.